meson.build

```diff
@@ -61,6 +61,34 @@ executable(
 	],
 	include_directories: [inc],
 	install: true,
+	install_dir: get_option('libexecdir'),
```

Comment:
Shouldn't this use `prefix` as well?

Reply:
No, `install_dir` is relative to `prefix`.

Reply:
Ah, indeed!